src/server/input.ts

```diff
@@ -57,7 +57,15 @@ export class Keyboard {
     if (kModifiers.includes(description.key as types.KeyboardModifier))
       this._pressedModifiers.add(description.key as types.KeyboardModifier);
     const text = description.text;
-    await this._raw.keydown(this._pressedModifiers, description.code, description.keyCode, description.keyCodeWithoutLocation, description.key, description.location, autoRepeat, text);
+    await Promise.all([
+      this._page.context().notifyInputListeners(this._page, {
```

Comment:
This now conflicts with both pw:api and trace viewer. I wish we could have it all aligned.